app/src/pages/prompt/PromptVersionsList.tsx

```diff
@@ -35,10 +37,15 @@ const promptVersionItemCSS = css`
  * add /:versionId to the current path
  */
 export const PromptVersionItem = ({ version }: PromptVersionItemProps) => {
+  const { versionId } = useParams();
   const navigate = useNavigate();
+  const active = versionId === version.id;
```

Comment:
not crazy important since this is probably the only place this will be used but it might make more sense for the `isActive` to be a prop if the thought is to make this re-usable. That way the routing sits higher up and is co-located with other param parsing. Not blocking but I tend to like the routing hooks to be not super nested.